docs/cookbook/creating_landing_page_blocks_(enterprise).md

```diff
@@ -263,163 +240,60 @@ YUI.add('ezs-addcustomblockplugin', function (Y) {
-     *
-     * @param array $attributes
-     *
-     * @throws \EzSystems\LandingPageFieldTypeBundle\Exception\InvalidBlockAttributeException
-     */
-    public function checkAttributesStructure(array $attributes)
-    {
-        if (!isset($attributes['content'])) {
-            throw new InvalidBlockAttributeException('Tag', 'content', 'Content must be set.');
-        }
-    }
-}
-```
-
-!!! tip
-
-    If you want to make sure that your block is only available in the Element menu in a specific situation, you can override the `isAvailable` method, which makes the block accessible by default:
-
-    ``` php
-    public function isAvailable()
-        {
-            return true;
-        }
-    ```
-
-#### service.yml configuration
-
-``` yaml
-# services.yml
-ezpublish.landing_page.block.tag:
-    class: EzSystems\LandingPageFieldTypeBundle\FieldType\LandingPage\Model\Block\TagBlock
-    tags:
-        - { name: landing_page_field_type.block_type, alias: tag }
-```
-
-#### Block template
-
-`{{ block.attributes.content|raw }}`
-
 ## Defining Landing Page blocks in the configuration file
 
-There is an alternative way to define/configure Landing Page block types that requires modifying the `config.yml` in an application or a bundle and using the `ez_systems_landing_page_field_type` key.
+There is a faster and more developer-friendly way to create Landing Page block types
+using only the YAML configuration in an application or a bundle, under the `ez_systems_landing_page_field_type` key.
 
 ``` yaml
-# example.yml
 ez_systems_landing_page_field_type:
     blocks:
-        new_block_name:
+        example_block:
             initialize: true
-            name: New Block Name
+            name: Example Block
             category: default
             thumbnail: bundles/ezsystemslandingpageblocktemplate/images/templateblock.svg
             views:
                 default:
-                    template: EzSystemsLandingPageBlockTemplateBundle::template.html.twig
+                    template: blocks/template.html.twig
                     name: Default view
+                special:
+                    template: blocks/special_template.html.twig
+                    name: Special view
             attributes:
-                text_field:
+                handle:
                     type: text
+                    regex: /[\s]/
+                    regexErrorMessage: Invalid text
                     required: true
                     inline: false
                     values: []
                     options: []
-                width:
+                flavor:
                     type: multiple
-                    regex: /[\s]/
-                    regexErrorMessage: Invalid text
                     required: true
                     inline: false
                     values: [value2]
                     options:
-                        value1: label1
-                        value2: label2
+                        value1: vanilla
```

Comment:
What are this flavors? Is it a CakeBlock?

Reply:
This is just an example, I figured it's more illustrative than simply "label1" and "label2"

Reply:
No, `regexErrorMessage` refers to `handle`, not `flavor`